specs/latest/2.0/index.html

```diff
@@ -4075,6 +4075,23 @@ <h3>blitFramebuffer rectangle width/height limitations</h3>
         blitFramebuffer is used to scale the framebuffer.
     </div>
 
+    <h3>Transform feedback buffer binding limitations</h3>
```

Comment:
There is already a section farther up about transform feedback buffer binding restrictions. I'd suggest moving this section next to the other one, or merging the sections together.

Reply:
There might be references to section numbers floating around, so it's better to add new sections to the end of the document. But it seems reasonable to merge the sections, I added another commit to this pull request that does just that.